roles/cs.aws-magento-cron/tasks/main.yml

```diff
@@ -27,6 +27,7 @@
         name: magento-cron-aws
         state: stopped
         enabled: no
+      ignore_errors: yes
```

Comment:
Ugly. Can't we make it nicer?
Why this is even executed for non-aws?

Reply:
Shouldn't this *whole role* be skipped when not on AWS?

PS `ignore_errors` is *evil* - it should be used only if you're executing a command that you *expect* to fail in some cases and then compare the registered _result_ to get some information.